src/source/generator.scala

```diff
@@ -114,6 +114,12 @@ package object generatorTools {
   }
   def q(s: String) = '"' + s + '"'
   def firstUpper(token: String) = if (token.isEmpty()) token else token.charAt(0).toUpper + token.substring(1)
+  def firstUpperOrID(token: String) = {
+    if (token == "id") "ID"
```

Comment:
Do we need to skip the first token? e.g. `id_xxx`, or it doesn't really matter as `id` only means `id`

Reply:
We don't apply this to the first word. So should be fine.

```scala
    val camelLowerID = (s: String) => {
      val parts = s.split('_')
      parts.head + parts.tail.map(firstUpperOrID).mkString
    }
```